src/main/kotlin/app/RepoHasher.kt

```diff
@@ -0,0 +1,89 @@
+// Copyright 2017 Sourcerer Inc. All Rights Reserved.
+// Author: Anatoly Kislov (anatoly@sourcerer.io)
+
+package app
+
+import app.model.Commit
+import app.model.Repo
+import app.utils.RepoHelper
+import io.reactivex.Observable
+import org.eclipse.jgit.api.Git
+import org.eclipse.jgit.revwalk.RevCommit
+import org.eclipse.jgit.revwalk.RevWalk
+import java.io.File
+import java.io.IOException
+
+/**
+ * RepoHasher hashes repository and uploads stats to server.
```

Comment:
My point was that "RepoHasher hashes repository" is not that descriptive as it could be, I would prefer to use 'analyze' word if it's indeed a synonym of 'hashes', something like "RepoHasher analyzes a repository, extract the stats and uploads it to the server"